### PR TITLE
Convert stats to read from Kinesis

### PR DESCRIPTION
**Pull request by marten**

### CODE REVIEW

File: bin/start_stream [hunk 1]
```diff
@@ -28,15 +28,6 @@ outputs = [
 processor = Stats::Processor.new(outputs)
 
 kafka_config = Stats::Config.load_yaml('kafka.yml', environment)
```
Comment (camallen) [inline bin/start_stream]: should this go to?

File: lib/input/kcl_reader.rb [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+module Stats
+  module Input
+    class KclReader < Aws::KCLrb::RecordProcessorBase
+      attr_reader :processor
+
+      def initialize(processor=$stderr)
+        @processor = processor
+      end
+
+      def init_processor(shard_id)
```
Comment (camallen) [inline lib/input/kcl_reader.rb]: empty method or is this needed?

File: lib/processor.rb [hunk 3]
```diff
@@ -7,61 +10,8 @@ def initialize(outputs)
...
-          longitude: match.longitude
-        })
-      else
-        event
-      end
-    rescue StandardError
-      event
-    end
-
-    def strip_private_fields(event)
```
Comment (camallen) [inline lib/processor.rb]: I can't see where we're stripping the IP in the output messages?

File: lib/input/kcl_reader.rb (hunk 2 above)
Comment (marten) [inline lib/input/kcl_reader.rb]: The base class raises a NotImplemented error.

File: bin/start_stream (hunk 1 above)
Comment (marten) [inline bin/start_stream]: Nah, we can get rid of that.

File: lib/processor.rb (hunk 3 above)
Comment (marten) [inline lib/processor.rb]: The models are now whitelisting/transforming the event data.

File: bin/start_stream (hunk 1 above)
Comment (marten) [inline bin/start_stream]: The AWS gem can just load credentials from the ENV btw.

File: Rakefile [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+
+  puts "Running the Kinesis sample processing application..."
+  classpath = FileList["#{JAR_DIR}/*.jar"].join(':')
+  classpath += ":#{SAMPLES_DIR}"
+  ENV['PATH'] = "#{ENV['PATH']}:#{SAMPLES_DIR}"
+  commands = %W(
+    #{ENV['JAVA_HOME']}/bin/java
+    -classpath #{classpath}
+    -DinitialPositionInStream=LATEST
+    -DapplicationName=martentest1
```
Comment (marten) [inline Rakefile]: Missed this hardcoded value. Will fix.

**Comment by camallen** [comment]
LGTM after the hardcoded value.

**Pull request merged**